source/Preferences.cpp

```diff
@@ -222,6 +225,8 @@ void Preferences::Load()
 			previousSaveCount = max<int>(3, node.Value(1));
 		else if(node.Token(0) == "alt-mouse turning")
 			settings["Control ship with mouse"] = (node.Size() == 1 || node.Value(1));
+		else if(node.Token(0) == "notification settings")
+			dateFormatIndex = max<int>(0, min<int>(node.Value(1), NOTIF_OPTIONS.size() - 1));
```

Comment:
Should this really be dateFormatIndex for "notification settings"?

Reply:
sloppy copy-paste. gonna fix it